cli/azd/pkg/environment/environment.go

```diff
@@ -51,6 +51,10 @@ type Environment struct {
 	// dotenv is a map of keys to values, persisted to the `.env` file stored in this environment's [Root].
 	dotenv map[string]string
 
+	// dotEnvDeleted keeps track of deleted keys to be reapplied before a merge operation
+	// happens in Save
+	dotenvDeleted map[string]struct{}
```

Comment:
Similar to other comment, can we just call this `deletedItems` or `deletedValues` ?

Reply:
This I'll fix now since it's local.